apps/sap-commerce-cloud/frontend/README.md

```diff
@@ -6,15 +6,15 @@ In the project directory, you can run:
 
 ### Install packages
 
-`npm i` 
+`npm i`
 
 ### Set development ENV
 
 Copy the contents of .env.development.sample into a new file called .env.development (calls to SAP will only work from localhost with this added)
```

Comment:
Is this still true? I think we can remove this

Reply:
I wasn't sure! I'll verify and remove.